modules/document_repository/php/files.class.inc

```diff
@@ -119,13 +119,13 @@ class Files extends \NDB_Page
             $record = urldecode(basename($request->getUri()->getPath()));
             if (!is_numeric($record)) {
                 $file = __DIR__ . "/../user_uploads/$name/$record";
+                readfile($file);
                 return (new \LORIS\Http\Response())
-                ->withHeader(
-                    'Content-Disposition',
-                    'attachment; filename=' . basename($file)
-                )
-                ->withHeader('Content-Transfer-Encoding', 'Binary')
-                ->withBody(new \LORIS\Http\StringStream(readfile($file)));
```

Comment:
I'm pretty sure Zend Diactoros has a stream type for dealing with files (probably https://github.com/zendframework/zend-diactoros/blob/master/src/Stream.php), why not make a LORIS wrapper around similar to https://github.com/aces/Loris/blob/minor/src/Http/Response.php ?